docs/packaging/packaging-changes.md

```diff
@@ -9,42 +15,42 @@ This document is meant as a changelog of sorts for packaging practices. Refer to
 
 _These items are not mandatory yet, and are subject to change_
 
-#### `makefile` -> `go-task`
-
-- All packaging action are now handled by `go-task` rather than `make`.
-  - Example: `make` to build a package is replaced with `go-task build`.
-  - Most commands are otherwise the same: `make local`, for example, is replaced by `go-task local`.
-  - You can see all available commands at `common/Taskfile.yml`
-- Packagers should install `go-task` on their machines if they have not already
-- The `Makefile` included with every package is no longer required. Please delete it from a package when updating, and do not include it when making the initial commit of a new package
-
-#### `homepage` in `package.yml`
+#### Requiring `homepage` in `package.yml`
 
-- The `homepage` key in `package.yml`will become mandatory. If a package does not have a homepage, use the address for the upstream source.
+- The `homepage` key in `package.yml` will become mandatory. If a package does not have a homepage, use the address for the upstream source.
 - We are exploring how to enforce this; for now we will ask nicely.
 
 ## 2023
 
 ### September
 
+#### Switching From `make` to `go-task`
+
+- All packaging action are now handled by `go-task` rather than `make`.
+  - Example: `make` to build a package is replaced with `go-task`.
+  - Most commands are otherwise the same: `make local`, for example, is replaced by `go-task local`.
+  - You can see all available commands at `common/Taskfile.yml`
+- Packagers should install `go-task` on their machines if they have not already
+- The `Makefile` included with every package is no longer required. Please delete it from a package when updating, and do not include it when making the initial commit of a new package
+
 #### GitHub Monorepo Migration
 
-- Packages were moved from the multirepo setup to a monorepo: `github.com/getsolus/packages` Now packagers only need to clone/fork one repository for all packages.
+- Packages were moved from the multirepo setup to a monorepo: [github.com/getsolus/packages](https://github.com/getsolus/packages) Now packagers only need to clone/fork one repository for all packages.
```

Comment:
Here you use "multirepo" but later you use "mono-repo". They should be made to be consistent with one or the other.

Reply:
changed to singleword